tests/Acoustics.Test/TowseyLibrary/DataToolsTests.cs

```diff
@@ -38,5 +41,85 @@ public void TestConcatenateVectorsOverload()
 
             CollectionAssert.AreEqual(expected, actual);
         }
+
+        [TestMethod]
+        public void TestPlotImages()
+        {
+            var plots = new List<Plot>();
+
+            // Prepare and initialise three different plots. All linear but having different lengths.
+            //prepare plot 2
+            double[] array1 = new double[500];
+            for (int i = 0; i < 500; i++)
+            {
+                array1[i] = i;
+            }
+
+            string title1 = "Plot 1";
+            double threshold1 = 200.0;
+            Plot plot1 = Plot.PreparePlot(array1, title1, threshold1);
+            plots.Add(plot1);
+
+            //prepare plot 2
+            double[] array2 = new double[400];
+            for (int i = 0; i < 400; i++)
+            {
+                array2[i] = i;
+            }
+
+            string title2 = "Plot 2";
+            double threshold2 = 150.0;
+            Plot plot2 = Plot.PreparePlot(array2, title2, threshold2);
+            plots.Add(plot2);
+
+            //prepare plot 3
+            double[] array3 = new double[200];
+            for (int i = 0; i < 200; i++)
+            {
+                array3[i] = i;
+            }
+
+            string title3 = "Plot 3";
+            double threshold3 = 80.0;
+            Plot plot3 = Plot.PreparePlot(array3, title3, threshold3);
+            plots.Add(plot3);
+
+            int plotHeight = 50;
+
+            // now concatenate the plots WITHOUT rescaling their length.
+            var imageList1 = new List<Image<Rgb24>>();
+            foreach (var plot in plots)
+            {
+                var image = plot.DrawAnnotatedPlot(plotHeight);
+                imageList1.Add(image);
+            }
+
+            // create the image for visual confirmation
+            var concatImage1 = ImageTools.CombineImagesVertically(imageList1);
+            concatImage1.Save(this.TestOutputDirectory + "ConcatImage1.png");
```

Comment:
Do not join paths with the `+` operator.

Use the `SaveTestOutput` function

This comment applies to all usages in this class.